lib/exposure/location.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:covidtrace/exposure/exposure.dart';
+import 'package:covidtrace/helper/datetime.dart';
+import 'package:covidtrace/storage/location.dart';
```

Comment:
I've just been using relative paths for files in our package. Seems like vs code's behavior is inconsistent when auto importing.

Reply:
I think package scoped imports seem more common in other Dart projects - in fact it's nice so if a file that imports ten other project packages is moved to a subdirectory you don't have to update any of the import paths.